core/google/api/core/gapic_v1/method.py

```diff
@@ -33,6 +33,7 @@
 _API_CORE_VERSION = pkg_resources.get_distribution('google-cloud-core').version
 METRICS_METADATA_KEY = 'x-goog-api-client'
 USE_DEFAULT_METADATA = object()
+DEFAULT = object()
```

Comment:
Want to give this value a docstring so you can refer to it?

Reply:
Done.